src/main/groovy/FrontEndTestJobBuilder.groovy

```diff
@@ -0,0 +1,42 @@
+import JsJobBuilder
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+class FrontEndTestJobBuilder {
+
+    String name
+    String description
+    String gitBranch = 'master'
+    String pollScmSchedule = '@daily'
```

Comment:
We have 2 options:
1. Delete them
2.  The job itself is pretty skimpy. We can functionality that leverages most of these fields. Will delete the rest then?